pkg/project/version_bundle.go

```diff
@@ -8,23 +8,8 @@ func NewVersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "calico",
-				Description: "Updated from v3.8.2 to v3.9.1.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "containerlinux",
-				Description: "Updated from v2135.4.0 to v2191.5.0.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "etcd",
-				Description: "Updated from v3.3.13 to v3.3.15.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "kubernetes",
-				Description: "Updated from v1.14.6 to v1.15.5.",
+				Component:   "kvm-operator",
+				Description: "Modified to retrieve component versions from releases",
```

Comment:
This is unused - fine as a placeholder but we need a `CHANGELOG.md`

Reply:
Added Changelog